server/routes.py

```diff
@@ -27,4 +27,15 @@ def tag_by_id(tag_id):
     if request.method == 'DELETE':
         delete_tag_response = database.delete_tag_by_id(conn, tag_id)
         return delete_tag_response
-        
\ No newline at end of file
+
+# subscribers routes the /tags/:id/subscribers endpoint
+@app.route('/tags/<int:tag_id>/subscribers', methods=['GET','POST'])
+def subscribers(tag_id):
+    if request.method == 'GET':
+        get_subscribers_response = database.read_subscribers(conn, tag_id)
+        return get_subscribers_response
+    if request.method == 'POST':
+        body = request.get_json()
+        subscriber_id = body['subscriber_id']
```

Comment:
Can we check if no Id was provided and return a proper response for bad request?